music/src/core/data.ts

```diff
@@ -370,8 +384,7 @@ export class MelodyConverter extends DataConverter {
     super(args);
     this.minPitch = args.minPitch;
     this.maxPitch = args.maxPitch;
-    this.ignorePolyphony =
-        (args.ignorePolyphony === undefined) ? true : args.ignorePolyphony;
+    this.ignorePolyphony = args.ignorePolyphony;
```

Comment:
does this change the behavior?

Reply:
No, because the Melody code now uses true as the default value for ignorePolyphony.